tips.json

```diff
@@ -30,5 +30,23 @@
     "tool": "OS X",
     "toolSlug": "osx",
     "text": "[cmd + tab] will help you qickly cycle through your open applications instead of having to click on them from the dock. So fast!"
+  },
+  {
+    "tool": "EditorConfig",
+    "toolSlug": "editors",
+    "source": "http://editorconfig.org/",
+    "text": "You can use the .editorconfig format to maintain consistent coding styles between different editors and IDEs. Excellent if you are working on a group!"
+  },
+  {
+    "tool": "Git",
+    "toolSlug": "gitconfig",
```

Comment:
Let's make the toolSlug for this just "git". I imagine us having a lot of git tips and I'd rather start a little more generic while it's still tiny.